manifests/host.pp

```diff
@@ -1,17 +1,13 @@
 # == Define: dhcp::host
 #
 define dhcp::host (
-  $ip,
-  $mac,
-  $options = {},
-  $comment='',
-  $ignored = false,
+  Stdlib::Compat::Ipv4 $ip,
```

Comment:
I recall this module could also be used to manage a dhcpv6 server so I wonder if you're getting overly strict here.